src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -823,6 +827,15 @@ $kCodeBlockDarkLineMatchBackgroundColor: rgba(255, 255, 255, 0.12); // we don't
     padding: var(--kui-space-40, $kui-space-40);
     position: relative;
 
+    @media (min-width: $kui-breakpoint-laptop) {
+      &:hover .code-block-secondary-actions {
+        .secondary-actions,
+        :deep( .code-block-copy-button) {
+          opacity: 1;
+        }
+      }
+    }
```

Comment:
Oh, I may not have explained it clearly earlier. What I meant was to suggest merging the two media queries into one and defining it in the section below, ensuring that related styles are neatly organized together.